Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.9-slim
+
+ENV LECTURECUT_HOME=/LectureCut
+WORKDIR ${LECTURECUT_HOME}
+COPY . .
```

Comment:
With the recent changes this should copy from the `src` folder. (This somewhat contradicts my previous statement regarding the `.dockerignore` file. Feel free to remove it again.)

Reply:
Ah ok. I made adjustments to Dockerfile and removed the dockerignore.